src/test/java/ca/uhn/fhir/jpa/starter/ExampleServerR5IT.java

```diff
@@ -99,7 +114,8 @@ public void testWebsocketSubscription() throws Exception {
     SocketImplementation mySocketImplementation = new SocketImplementation(mySubscriptionId.getIdPart(), EncodingEnum.JSON);
 
     myWebSocketClient.start();
-    URI echoUri = new URI("ws://localhost:" + port + "/websocket");
+
+	  URI echoUri = new URI(endpoint);
```

Comment:
General comment:  It seems our formatting (tabs) conflcts with jpa-serverstarter's formatting (spaces).   It looks like your changes use tabs.